wgpu-core/src/instance.rs

```diff
@@ -453,11 +458,11 @@ impl<B: GfxBackend> Adapter<B> {
         // Check features that are always needed
         let wishful_features = hal::Features::ROBUST_BUFFER_ACCESS
             | hal::Features::VERTEX_STORES_AND_ATOMICS
-            | hal::Features::FRAGMENT_STORES_AND_ATOMICS
             | hal::Features::NDC_Y_UP
             | hal::Features::INDEPENDENT_BLENDING
             | hal::Features::SAMPLER_ANISOTROPY
-            | hal::Features::IMAGE_CUBE_ARRAY;
+            | hal::Features::IMAGE_CUBE_ARRAY
+            | hal::Features::SAMPLE_RATE_SHADING;
```

Comment:
These all should likely be candidates for downlevel features. Don't need to do anything about these now, I'll file an issue to follow up.